app/controllers/api/v1/organizations_controller.rb

```diff
@@ -14,6 +14,7 @@ def public
                                                   image
                                                   phone
                                                   address
+                                                  organization_slides
```

Comment:
In the case you want to show the slides that belong to an organization, there is an has_many method for the serializer.